File: components/example/component/applications/prudence-example/libraries/resources/python/person.py

class Person:
    def handle_init(self, conversation):
        conversation.addMediaTypeByName('text/html')
        conversation.addMediaTypeByName('text/plain')
        
    def handle_get(self, conversation):
        id = conversation.locals['id']
        return 'I am person %s, formatted as "%s", encased in Python'  % (id, conversation.mediaTypeName)

File: components/example/component/applications/prudence-example/libraries/resources/php/person.php
<?php

class Person {
	function handle_init($conversation) {
		$conversation->addMediaTypeByName('text/html');
		$conversation->addMediaTypeByName('text/plain');
	}

	function handle_get($conversation) {
		$id = $conversation->locals['id'];
		return "I am person {$id}, formatted as \"{$conversation->mediaTypeName}\", encased in PHP";
	}
}

?>